integration-cli/docker_api_stats_test.go

```diff
@@ -231,6 +231,9 @@ func (s *DockerSuite) TestApiStatsContainerNotFound(c *check.C) {
 func (s *DockerSuite) TestApiStatsContainerGetMemoryLimit(c *check.C) {
 	testRequires(c, DaemonIsLinux)
```

Comment:
Check out https://github.com/docker/docker/blob/master/integration-cli/requirements_unix.go , it looks like you can just add one of the memory ones to the testRequires here.